frontend/server/src/SecurityTools.php

```diff
@@ -302,4 +302,56 @@ public static function getCourseCloneAuthorizationToken(
             ->setClaims($claims);
         return $token->toString();
     }
+
+    public static function getDecodedCloneCourseToken(
+        string $sharedKey
```

Comment:
```suggestion
        string $token
```

lo que se está proveyendo a esta función es un token, no una llave.

Reply:
Listo